config/samples/smoke-dist/requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy>=1.16.3
+Cython>=0.29.4
+requests>=2.21.0
+urllib3>=1.21.1
+pandas>=0.23.2
```

Comment:
pandas (and maybe others) doesn’t seem used.

Reply:
I would pandas later for transform the input data into dmatrix. thanks terry.

Reply:
Then I'd suggest adding them later when needed.